resources/test/fixtures/B024.py

```diff
@@ -0,0 +1,129 @@
+"""
+Should emit:
+B024 - on lines 17, 34, 52, 58, 69, 74, 84, 89
+"""
+
+import abc
+import abc as notabc
+from abc import ABC, ABCMeta
+from abc import abstractmethod
+from abc import abstractmethod as abstract
+from abc import abstractmethod as abstractaoeuaoeuaoeu
+from abc import abstractmethod as notabstract
+
+import foo
+
+
+class Base_1(ABC):  # error
+    def method(self):
+        foo()
+
+
+class Base_2(ABC):
+    @abstractmethod
+    def method(self):
+        foo()
+
+
+class Base_3(ABC):
+    @abc.abstractmethod
+    def method(self):
+        foo()
+
+
+class Base_4(ABC):
+    @notabc.abstractmethod
+    def method(self):
+        foo()
```

Comment:
Import tracking doesn't support aliases yet. https://github.com/charliermarsh/ruff/issues/720

Reply:
(Will be fixed today, hopefully.)

Reply:
Thanks!